fdbserver/storageserver.actor.cpp

```diff
@@ -1773,6 +1773,8 @@ ACTOR Future<Standalone<RangeResultRef>> tryGetRange( Database cx, Version versi
 	if( *isTooOld )
 		throw transaction_too_old();
 
+	cx->minAcceptableReadVersion =
+	    std::min(cx->minAcceptableReadVersion, version); // Suppress minAcceptableReadVersion check
```

Comment:
Is the idea here that the process will never switch clusters because it's a server process, and therefore we don't have to worry about it trying to read versions based on another cluster?

This seems like it works, though it's maybe not clear from looking at it that this suppression doesn't just apply locally but instead to `cx` in general. Maybe a more transparent way to do this is to support opening the `Database` in an unswitchable mode, which would both disallow switching the connection file and eliminate the version restriction.

Reply:
The reason this line is necessary is that this `cx` does not perform a GetReadVersion prior to this read. Opening `Database` in an unswitchable mode makes sense, I'll get started on that.